tuxemon/event/actions/spawn_monster.py

```diff
@@ -53,15 +53,15 @@ def start(self) -> None:
         mother_id = uuid.UUID(trainer.game_variables["breeding_mother"])
```

Comment:
I know you didn't change line 51, but could you get rid of that assert too while you're in here?